src/id3/v2/tag.rs

```diff
@@ -872,10 +915,56 @@ impl<'a> Id3v2TagRef<'a, std::iter::Empty<FrameRef<'a>>> {
 
 // Create an iterator of FrameRef from a Tag's items for Id3v2TagRef::new
 pub(crate) fn tag_frames(tag: &Tag) -> impl Iterator<Item = FrameRef<'_>> + Clone {
+	fn create_frameref_for_number_pair<'a>(
+		number: Option<&str>,
+		total: Option<&str>,
+		id: &'a str,
+	) -> Option<FrameRef<'a>> {
+		let mut value: Option<Cow<'_, str>> = None;
+
+		if let Some(number) = number {
+			value = Some(Cow::Borrowed(number));
+		}
+
+		if let Some(total) = total {
+			let number = value.unwrap_or_else(|| Cow::Owned(DEFAULT_NUMBER.to_string()));
+
+			value = Some(Cow::Owned(merge_num_pair(number, total)));
+		}
+
+		value.map(|value| {
+			let frame = Frame::text(Cow::Borrowed(id), value.to_string());
+
+			FrameRef {
+				id: frame.id,
+				value: Cow::Owned(frame.value),
+				flags: frame.flags,
+			}
+		})
+	}
+
+	let number_pair_keys = &[
```

Comment:
This should be extracted as a constant of the module. Probably near the other _number_ constants. Serves as additional documentation.

Reply:
I fixed this part.